test/linalg/schur.jl

```diff
@@ -42,6 +42,8 @@ aimg  = randn(n,n)/2
         @test vecs*sch*vecs' ≈ asym
         sch, vecs, vals = schur(Symmetric(a+a.'))
         @test vecs*sch*vecs' ≈ a + a.'
+        sch, vecs, vals = schur(Tridiagonal(a+a.'))
+        @test vecs*sch*vecs' ≈ AbstractArray(Tridiagonal(a + a.'))
```

Comment:
The `AbstractArray` seems unnecessary? :)

Reply:
Is it? I think the types won't match? Or are you saying isapprox will do the convert for us?

Reply:
The latter :).

Reply:
Nice, I'll edit it then. Thanks for catching that!